sites/mundopmmi/server/templates/content/index.marko

```diff
@@ -49,6 +49,7 @@ $ const displayRailRelatedContent = false;
         <if(displayRailAds)>
           <endeavor-gam-ad-unit-display id="gpt-ad-rail1" />
         </if>
+        <div id="gtm-content-rail1"></div>
```

Comment:
No ad unit slots have been defined for any of these ads. They'll never be able to traffic anything...